ckanext/canada/tables/qpnotes.yaml

```diff
@@ -0,0 +1,357 @@
+dataset_type: qpnotes
+target_dataset: qpnotes
+
+title: Proactive Disclosure - Question Period Notes
+notes: Access, upload and modify Question Period notes for your organization
+
+template_version: 3
+template_updated: 2019-11-18
+
+portal_type: info
+collection: qpnotes
```

Comment:
The collection here corresponds to the dataset collection for search purposes https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/schemas/presets.yaml#L33 These have been set to "pd" for the most part. Odd, though that I don't have that in the choice list.